### DISCUSSION TITLE
New fonts and icons proposals

### DISCUSSION THREAD

**Discussion by Finii** [ryanoasis/nerd-fonts#1095] [Ideas]
All the new fonts and icon set request get confusing in the Issues section.

The requests are set to 'closed' to make them vanish from the Issues (so we can concentrate on real issues there), and get a link in the appropriate section below.

**Please lets keep this discussion very tidy.**

**Very tidy means almost no comments. No discussions, this is an overview. Please use Issues or dedicated Discussions.**
**Comments might be deleted without explanation.**

You should create an Issue in any case, so that we have a place to discuss just that one item.
Here is the meta topic that just collects them all in one place.

If you like to upvote something do so in the linked Issue and not here. The Issues will all be examined despite the fact that they are marked closed.

**Comment by Finii** [comment]
# New fonts to be added to `patched-fonts`
Every font added will of course add some maintenance burden and increases the repo size until we have proper separation.

* #300
* #314 
* #350 
* #383 
* #386 
* #399 
* #548 
* #587 
* #972 
* #995 
* #1011
* #1299
* #1335 
* #1458
* #1475
* #1512
* #1533
* #1607
* #1641
* #1650
* #1658
* #1685
* #1759 
* #1776 
* #1799
* #1817
*
* #1812 
* #1886 
* #1889  
* #1895 
* #1905 
* #1911
* #1933 
* #1987 
* #1995 
* #2002

**Comment by Finii** [comment]
# Add glyph to core set
We can add glyphs to the `original-source` set, i.e. `nf-custom-` IDs.
But the number of codepoints is limited. Usually better is to put the glyphs in one of the upstream glyph collections.

* #360 
* #851 
* #1135
* #1196 
* #1237
* #1447
* #1474
* #1484
* #1590 
* #1672
* #1715
* #1730
* #1796
* #1824
* #1825
* #1826
* #1832
* #1833
* #1835 
* #1891
* #1893 
* #1920 
* #1951 
* #1986
* #2005 
* #2025

**Reply by okkdev** [reply]
I've added a comment to the Gleam icon PR 🙂 https://github.com/ryanoasis/nerd-fonts/pull/1591#issuecomment-2970384336

**Reply by H4ckN3rd** [reply]
Hi! Just wanted to ask if there’s any update on when the Obsidian logo glyph might be added to Nerd Fonts?

It's a pretty widely-used app among devs and productivity users, and having its icon available as a nf-* glyph would be super helpful for status bars, Zebar/Waybar, terminals, etc.  https://github.com/ryanoasis/nerd-fonts/issues/1672

**Comment by Finii** [comment]
# New glyphs sets
Complete sets of symbols. This might need codepoint shuffling.

* #153 
* #173 
* #233 
* #616 
* #655 
* #665
* #1301
* #1389
* #1448
* #1449
* #1606
* #1791
* #1808
* #1814
* #1871 
* #1878 
* #1885 
* #1957
* #1959 
* #1967

**Reply by hpfr** [reply]
Thanks for consolidating these. I believe my comments (https://github.com/ryanoasis/nerd-fonts/issues/233#issuecomment-994202220) about [File Icons](https://github.com/file-icons) still apply, particularly to #233 and #615

**Comment by Finii** [comment]
# Replacement glyph sets
Replace one set with a more maintained one.

* #465

**Comment by moebiusband73** [comment]
Hi,
not sure if this belongs here. I ran into an issue that some Icons were not shown for an  identical Terminal with identical patched font file on a a different OS. After some debugging it turned out that a bunch of icons used (in my case in Neovim with plugins) used glyphs that are not resolved by the patched (even complete) Nerdfont. These are specifically the Dingbats Symbolset (used for arrows) and the Apple Braille set (used as animated wait icon sequence). It depends on the terminal emulator and OS if the missing glyphs are resolved through fallback fonts. There are tickets (I found one in Kitty and others in various Neovim plugins) where workarounds for this issue were introduced. A clean solution would be that those (standard) symbol sets are by default included in the Nerdfont patchset. Of course one can create your own patched font including those symbol fonts, but due to the not obvious reason many are not aware what the real cause for their problems is.
